controllers/depresolver/depresolver_test.go

```diff
@@ -603,15 +603,39 @@ func TestResolveInvalidExtGeoTags(t *testing.T) {
 	}
 }
 
-func TestResolveGeoTagExistsWithinExtGeoTags(t *testing.T) {
+func TestResolveOnlyGeoTagExistsWithinExtGeoTags(t *testing.T) {
 	// arrange
 	defer cleanup()
 	tag := "us-west1"
 	expected := predefinedConfig
 	expected.ClusterGeoTag = tag
-	expected.ExtClustersGeoTags = []string{"us-east1", tag}
-	// act,assert
-	arrangeVariablesAndAssert(t, expected, assert.Error)
+	expected.ExtClustersGeoTags = []string{}
+	configureEnvVar(expected)
+	os.Setenv(ExtClustersGeoTagsKey, tag)
```

Comment:
`configureEnvVar(expected)` maps the expected entity to envvars, you can use something like this:

```go
	expected.ExtClustersGeoTags = []string{ExtClustersGeoTagsKey}
	configureEnvVar(expected)
```

Please can you change in all tests ?

Reply:
but the expected value should be an empty array, no? 
The idea is that we set the `EXT_GSLB_CLUSTERS_GEO_TAGS` to `tag `, `ClusterGeoTag` is also = `tag` => `config. ExtClustersGeoTags` should be empty. Test is quite artificial, perhaps I should change it in a way that at least one tag will remain in `ExtClustersGeoTags`, wdyt?

 same thing as it https://github.com/k8gb-io/k8gb/pull/750#discussion_r753044252

the current `configureEnvVar(expected) & NewDependencyResolver().ResolveOperatorConfig()` combo is not designed on this situation where we "change" the value during the validation process

Reply:
sorry, menat it this :

```go
	expected.ExtClustersGeoTags = []string{tag}
	configureEnvVar(expected)
```